### PR TITLE
EG-4 Fix

### PR DESCRIPTION
**Pull request by Icepicked**
## About the PR

Updated the EG-4 Energy Revolver:
- Fixed a spelling mistake.
- Minor recharge buff.

## Media

image

## Requirements

- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- tweak: Slight recharge buff to EG-4.
- fix: Spelling mistake on the EG-4.

## Summary by CodeRabbit

- **Bug Fixes**
	- Corrected a typographical error in the description of the EG-4 energy revolver.

- **New Features**
	- Increased the auto-recharge rate of the BatterySelfRecharger component, enhancing weapon performance.

### CODE REVIEW

File: Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -89,5 +89,5 @@
     - Sidearm
   - type: BatterySelfRecharger
     autoRecharge: true
-    autoRechargeRate: 35
+    autoRechargeRate: 45
```
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:92]: 40 is the antique laser's recharge rate, having it on-par with that might be nice

**Comment by Aidenkrz** [comment]
> ## Walkthrough
> The pull request introduces two changes to the `battery_guns.yml` file concerning the `EG-4 energy revolver`. It corrects a typographical error in the weapon's description and increases the `autoRechargeRate` of the `BatterySelfRecharger` from 35 to 45. No alterations were made to the declarations of exported or public entities.
> 
> ## Changes
> File Path	Change Summary
> Resources/Prototypes/Goobstation/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml	Corrected "advabced" to "advanced" in the weapon description; increased `autoRechargeRate` from 35 to 45.
> ## Possibly related PRs
> * **[EG-4 Energy Revolver #653](https://github.com/Goob-Station/Goob-Station/pull/653)**: The changes in this PR involve the `EG-4 energy revolver`, which is directly related to the modifications made to the `battery_guns.yml` file in the main PR, as both involve the same weapon and file.
> 
> ## Suggested labels
> `Changes: Sprites`, `No C#`
> 
> ## Suggested reviewers
> * whateverusername0
> * Piras314
> 
> ## Poem
> > In the land of code, a change so bright,
> > A typo fixed, now feels just right.
> > The energy flows, the battery's might,
> > With numbers adjusted, it shines in the night.
> > Hooray for the guns, let’s hop with delight! 🐰✨
> 
> Recent review details
> Thank you for using CodeRabbit. We offer it for free to the OSS community and would appreciate your support in helping us grow. If you find it useful, would you consider giving us a shout-out on your favorite social media?
> 
> Share
> Tips
> Tip
> 
> Early access features: enabled

holy shit this is like, actually helpful for once

**Comment by Piras314** [comment]
> Hooray for the guns, let’s hop with delight! 🐰✨

**Comment by whateverusername0** [comment]
@Icepicked you still have missing locales in mode selection

**Comment by Piras314** [comment]
@Icepicked the conflicts are because of the great rename (the file should be in _Goobstation not Goobstation) also look at whateverusername's comment

**Comment by Icepicked** [comment]
guh

**Comment by Icepicked** [comment]
@whateverusername0 @Piras314 im kinda dog at github, gonna add all the stuff in a new PR because this one was before the great rename.

**Comment by Piras314** [comment]
@Icepicked fair, use the diff of this PR to make the new one so it's easier.

**Pull request closed without merging**